plugins/css-prefers-color-scheme/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changes to Prefers Color Scheme
 
+### Unreleased (major)
+
+- Breaking : removed old CDN urls
+- Breaking : remove `color-depth` queries fallback
+- Breaking : remove 'no-preference' support as this was dropped from the spec
+- Breaking : remove old global object
```

Comment:
Very much nit but keeps consistent with this and all of the other CHANGELOG files

```suggestion
- Breaking: removed old CDN urls
- Breaking: remove `color-depth` queries fallback
- Breaking: remove 'no-preference' support as this was dropped from the spec
- Breaking: remove old global object
```

Reply:
Good catch!